### DISCUSSION TITLE
pandoc markdown to docx - cannot convert inline svg

### DISCUSSION THREAD

**Discussion by netsio** [jgm/pandoc#9849] [General]
I have markdown file which contains inline <svg> tag.
When converting markdown file to docx, the inline svg is not being generated in the docx. 
How to resolve this?

The markdown file content:
```
# Title

## Sample SVG
<svg height="100" width="100" xmlns="http://www.w3.org/2000/svg">
  <circle r="45" cx="50" cy="50" fill="red" />
  Sorry, your browser does not support inline SVG.  
</svg> 
 

```

**Comment by jgm** [comment]
The SVG tags are parsed as raw HTML and will be passed through to output formats that support that, but ignored in others, just like all raw HTML.
